src/reverb/views/templates/admin/tabs/sync_status.tpl

```diff
@@ -6,4 +6,48 @@
 * @license Apache License Version 2.0, January 2004
 * @package Reverb
 *}
+<div class="panel">
+<div class="row">
+    <div class="col-md-12 col-xs-12">
+        <div class="col-md-2 col-xs-2">
+                <span id="product_sync_action" class="button btn btn-default ">
+                  <span>{l s='Launch Listing Sync' mod='reverb'}</span>
+                </span><br />
+            <span class="pending-listing " style="display:none;">{l s='Pending Listing Sync...' mod='reverb'}</span>
+        </div>
+        <div class="col-md-10 col-xs-10">
+            <span>{l s='This manual action starts the cron job.' mod='reverb'}</span>
+        </div>
+    </div>
+</div>
+</div>
+
 {$reverb_sync_status}
+
+<script type="text/javascript">
+    $(document).ready(function() {
+        $('#product_sync_action')
+            .removeAttr('disabled') // Remove disabled attribute
+            .click (function(){
+                var select = $(this);
+                select.attr('disabled', 'disabled');
+                $('.pending-listing').show();
+                // Ajax call with secure token
+                $.get('{$url_site}/modules/reverb/cron.php?code=products',
+                    function (response) {
+                        showSuccessMessage("{l s='Listing Synced with success' mod='reverb'}");
+                        $('.pending-listing').hide();
+                        $(location).attr('href','{$module_url}');
```

Comment:
We changed the method through relead(true) because we use an ajax call when we launch the sync through the button. When the development was created the first time in the order status template, we noticed that it was necessary to reload the data of the table. We prefere to use Ajax call because the asynchronous processing is more performante